ReactAndroid/src/main/java/com/facebook/react/modules/accessibilityinfo/AccessibilityInfoModule.java

```diff
@@ -134,6 +141,44 @@ private void updateAndSendTouchExplorationChangeEvent(boolean enabled) {
     }
   }
 
+  // Converts a combination of bits into an array of strings.
+  // This is necessary when the assistive technology parameter has multiple flags.
+  private WritableArray convertBitsToStringArray(int bits, Function<Integer, String> convertToString) {
+    WritableArray strings = Arguments.createArray();
+    while (bits != 0) {
+      final int bit = (1 << Integer.numberOfTrailingZeros(bits));
+      strings.pushString(convertToString.apply(bit));
+      bits &= ~bit;
+    }
+    return strings;
+  }
+
+  private WritableArray createServicesArray(List<AccessibilityServiceInfo> accessibilityServiceList) {
```

Comment:
The process of receiving `List<AccessibilityServiceInfo>` and writing it to a JavaScript array is the same as #31200.
I added another method so that each other can use the same logic of AccessibilityServiceInfo.